test/versus_test.go

```diff
@@ -386,7 +389,7 @@ func hitsById(res *bleve.SearchResult) map[string]*search.DocumentMatch {
 		hit.Score = math.Trunc(hit.Score*1000.0) / 1000.0
 		hit.IndexInternalID = nil
 		hit.HitNumber = 0
-
+		hit.BytesRead = 0
```

Comment:
Again, why is this necessary?

Reply:
BytesRead doesn't relevance to that particular test, so I've disabled comparison over that. I've included comments for such settings.